src/main/java/mate/academy/springboot/swagger/service/ProductService.java

```
@@ -0,0 +1,41 @@
+package mate.academy.springboot.swagger.service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.swagger.model.Product;
+import mate.academy.springboot.swagger.repository.ProductRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProductService {
+    private final ProductRepository productRepository;
+
+    @Autowired
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    public Product save(Product product) {
+        return productRepository.save(product);
+    }
+
+    public Product getById(Long productId) {
+        return productRepository.getById(productId);
```

let's throw an exception in case product isn't found